cluster/config-defaults.yaml

```diff
@@ -896,13 +896,19 @@ stackset_routegroup_support_enabled: "true"
 # E.g. switching from RouteGroup to Ingress or vice versa.
 stackset_ingress_source_switch_ttl: "5m"
 
+# enable/disable configuration resources support for stackset
+stackset_configuration_rsc_enabled: "false"
```

Comment:
We need to enable this in the clusters where we already use the configmap feature.

Reply:
See also my comment below!